core/cli/src/lib.rs

```diff
@@ -237,6 +237,7 @@ where
 			"both" => service::ExecutionStrategy::Both,
 			"native" => service::ExecutionStrategy::NativeWhenPossible,
 			"wasm" => service::ExecutionStrategy::AlwaysWasm,
+			"nativeElseWasm" => service::ExecutionStrategy::NativeElseWasm,
```

Comment:
I think we should stick to all lowercase?

Reply:
yup